app/models/concerns/hyrax/user_override.rb

```diff
@@ -0,0 +1,57 @@
+require 'oauth'
+
+module Hyrax::UserOverride
```

Comment:
@VivianChu I am just curios whether this is a good practice to add a custom class in the Hyrax module or not. I think normally we only add those classes that are overridden from the upstream under the "hyrax" folder.

Reply:
@lsitu - This was something I was curious about. Essentially, as to whether there is a cleaner option for overriding modules/classes in Hyrax. It does seem quite significant to have to override/copy an entire module or class to add a single method, or update an existing one. For views/partials I can see it, but for classes and modules it seems like ideally there would be a cleaner object-oriented way to have overrides with a lighter footprint. So this issue [came up recently on the samvera google group](https://groups.google.com/forum/#!topic/samvera-tech/Z5bMJfPXZ1c) and @VivianChu was looking into it the feasibility of it.

Reply:
@mcritchlow Right. This is something that we often may need to spend sometime on for a better/cleaner way. But sometimes it could be challenge with the limitation of the hyrax design patterns that only allow us to do that way but not the other way ...